lib/grunt/plugins.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(grunt) {
 
 
   grunt.registerTask('docs', 'create angular docs', function() {
-    var gruntProc = shelljs.exec('"node_modules/.bin/gulp" --gulpfile docs/gulpfile.js');
+    var gruntProc = shelljs.exec('node "' + require.resolve('gulp/bin/gulp') + '" --gulpfile docs/gulpfile.js');
```

Comment:
I'd rather not depend on the internal `gulp` package structure. How about creating the npm script `gulp` in package.json:
```json
{
  "scripts": {
    "gulp": "gulp
  }
}
```
and then `npm run gulp -- --gulpfile docs/gulpfile.js` should work everywhere.